Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/DataReset.kt

```diff
@@ -66,7 +66,7 @@ class DataReset @Inject constructor(
     private val ratProfileSettings: RATProfileSettings,
     private val valueSetsRepository: ValueSetsRepository,
     private val vaccinationPreferences: VaccinationPreferences,
-    private val vaccinationRepository: VaccinationRepository,
+    private val vaccinationRepository: VaccinationRepository
```

Comment:
why? 
I think we will add more repos in the future here

Reply:
it wasn't intentional.
reverted